avalon/tools/cbloader/model.py

```diff
@@ -21,16 +21,25 @@ class SubsetsModel(TreeModel):
                "handles",
                "step"]
 
-    def __init__(self, parent=None):
+    SortAscendingRole = QtCore.Qt.UserRole + 2
+    sortDescendingRole = QtCore.Qt.UserRole + 3
```

Comment:
@davidlatwe only note - can we match the capital start? Either both `Sort` or `sort`? :)

Reply:
Well spot !! Fixed it :D